cpp/src/arrow/compute/api_scalar.h

```diff
@@ -57,19 +57,19 @@ class ARROW_EXPORT ElementWiseAggregateOptions : public FunctionOptions {
 enum class RoundMode : int8_t {
   /// Round to nearest integer less than or equal in magnitude (aka "floor")
   DOWN,
-  /// Round to nearest integer greater than or equal in magnitude (aka "ceil")
+  /// Round to nearest integer greater than or equal in magnitude (aka "ceiling")
```

Comment:
Hmm, "trunc" and "ceil" are common API names for this, for example [in C](https://pubs.opengroup.org/onlinepubs/9699919799/functions/ceil.html), [Python](https://docs.python.org/3/library/math.html#math.ceil) or [Rust](https://doc.rust-lang.org/std/primitive.f64.html#method.ceil), so perhaps keep those names here?

Reply:
Sure, I will make those changes.